src/index.jsx

```diff
@@ -8,17 +8,75 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Route, Switch } from 'react-router-dom';
 
-import Header, { messages as headerMessages } from '@edx/frontend-component-header';
+import { messages as headerMessages } from '@edx/frontend-component-header';
 import Footer, { messages as footerMessages } from '@edx/frontend-component-footer';
 
 import appMessages from './i18n';
-import ExamplePage from './example/ExamplePage';
+import { CoursePageResources } from './course-page-resources';
 import ProctoredExamSettings from './proctored-exam-settings/ProctoredExamSettings';
 import StudioHeader from './studio-header/Header';
 
 import './index.scss';
 import './assets/favicon.ico';
 
+// XXX this is just for testing and should be removed ASAP
+const coursePages = [
```

Comment:
Actually, lets move this into `CoursePageResources` (and not pass it in as a prop).

It's more suited there; we can init it with just a `courseId`, like `StudioHeader`.
We'll presumably be performing the API lookup from in the component, not in `index.jsx`, right?

Having as clean of an integration on this file (since it's core/common/shared by the live code) will minimize the risk of any issues/conflicts.